### PR TITLE
Add pull request templates

### PR DESCRIPTION
**Pull request by khalford**
Added pull request templates for ChatOps and Notebooks. Also, added a default pull request template with links to choose which template to use.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/notebook_pr_template.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+* [ ] Tested the image works?
+
+---
+
+### Reviewer:
+
+Have you done the following?:
+
+* [ ] Checked the `Dockerfile` is only installing the necessary packages?
+* [ ] Checked the permissions of downloaded files onto the container are appropriate?
```
Comment (meoflynn) [inline .github/PULL_REQUEST_TEMPLATE/notebook_pr_template.md:23]: Could we also add some checks for the reviewer:

- Does the `Dockerfile` switch to `NB_UID` at the end of the file? (i.e the image is run as user, **not** root)
- Has the image been built successfully and pushed to Harbor?

**Review by meoflynn — DISMISSED** [review]

File: .github/PULL_REQUEST_TEMPLATE/notebook_pr_template.md (hunk 1 above)
Comment (khalford) [reply .github/PULL_REQUEST_TEMPLATE/notebook_pr_template.md:23]: Done

File: .github/PULL_REQUEST_TEMPLATE/chatops_pr_template.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+-->
+
+---
+
+### Submitter:
+
+Have you done the following?:
+
+* [ ] Labeled the pull request from the following? `major | minor | patch` or `documentation | workflow`
+* [ ] Updated the documenation and template config files?
```
Comment (RJaliliSTFC) [inline .github/PULL_REQUEST_TEMPLATE/chatops_pr_template.md:14]: documentation typo

File: .github/PULL_REQUEST_TEMPLATE/chatops_pr_template.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+* [ ] Written integration tests?
+* [ ] Configured the Slack Application with new commands or permission scope changes?
+* [ ] Updated the `version.txt` and `docker-compose.yml` files?
+
+---
+
+### Reviewer:
+
+Have you checked the following?:
+* [ ] Is the version change is appropriate to the code changes? Semantic versioning documentation [here](https://semver.org/)
```
Comment (RJaliliSTFC) [inline .github/PULL_REQUEST_TEMPLATE/chatops_pr_template.md:28]: 2nd 'is' not necessary here

File: .github/PULL_REQUEST_TEMPLATE/chatops_pr_template.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+* [ ] Is the unit test code coverage is acceptable?
```
Comment (RJaliliSTFC) [inline .github/PULL_REQUEST_TEMPLATE/chatops_pr_template.md:29]: Another double is?

File: .github/PULL_REQUEST_TEMPLATE/notebook_pr_template.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+* [ ] Tested the image works?
+
+---
+
+### Reviewer:
+
+Have you done the following?:
+
+* [ ] Does the `Dockerfile` install only the necessary packages?
+* [ ] Are the permissions of downloaded files onto the container are appropriate?
```
Comment (RJaliliSTFC) [inline .github/PULL_REQUEST_TEMPLATE/notebook_pr_template.md:23]: double 'are'

**Review by RJaliliSTFC — APPROVED** [review]

**Pull request merged**